src/intro.c

```diff
@@ -1096,6 +1096,10 @@ static u8 SetUpCopyrightScreen(void)
         REG_DISPCNT = DISPCNT_MODE_0 | DISPCNT_OBJ_1D_MAP | DISPCNT_BG0_ON;
         SetSerialCallback(SerialCB_CopyrightScreen);
         GameCubeMultiBoot_Init(&gMultibootProgramStruct);
+    #if RHH_COPYRIGHT_INTRO == FALSE
+    case 1: // We need to overwrite REG_DISPCNT the second time, because the intro won't otherwise show on VBA.
+        REG_DISPCNT = DISPCNT_MODE_0 | DISPCNT_OBJ_1D_MAP | DISPCNT_BG0_ON;
+    #endif // RHH_COPYRIGHT_INTRO
```

Comment:
```suggestion
    case 1: // We need to overwrite REG_DISPCNT the second time, because the intro won't otherwise show on VBA.
```

Reply:
this is not an issue w.r.t. the intro, its just how it was discovered at first.

Reply:
Done